Content.Server/Pinpointer/PinpointerSystem.cs

```diff
@@ -22,6 +34,54 @@ public override void Initialize()
 
         SubscribeLocalEvent<PinpointerComponent, ActivateInWorldEvent>(OnActivate);
         SubscribeLocalEvent<FTLCompletedEvent>(OnLocateTarget);
+        SubscribeLocalEvent<AntagPinpointerComponent, GetVerbsEvent<Verb>>(OnGetVerb);
+    }
+
+    private void OnGetVerb(Entity<AntagPinpointerComponent> antagPinpointer, ref GetVerbsEvent<Verb> args)
+    {
+        if (!args.CanAccess || !args.CanInteract || args.Hands == null)
```

Comment:
Combine these, there is no reason to have this many separate if statements that all return the same thing